src/docs.md

```diff
@@ -9,11 +9,48 @@ description: The landing page for Flutter documentation.
         <div class="card-body">
             <header class="card-title">{{card.name}}</header>
             <p class="card-text">{{card.description}}</p>
-        </div>    
+        </div>
     </a>
 {% endfor %}
 </div>
 
+## What's new on flutter.io
+
+<b>Oct 31, 2018</b>
+
+Welcome to the revamped Flutter website!
+
+We've spent the last few months redesigning the website and how its
+information is organized. We hope you can more easily find the docs
+you are looking for. Some of the changes to the website include:
```

Comment:
I'd keep this list short, e.g., dropping any mention of the sitemap and 404.

Also, consider using "Redesigned" rather than "New" as a qualifier for most entries.